### PR TITLE
[KSE-1820] | Vedarth Sharma | Fixes ksql build failures

### PR DESCRIPTION
**Pull request by VedarthConfluent**
### Description 
Fixes ksql build pipeline

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")
- [ ] Do these changes have compatibility implications for rollback? If so, ensure that the ksql [command version](https://github.com/confluentinc/ksql/blob/master/ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/Command.java#L41) is bumped.

### CODE REVIEW

File: ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java [hunk 1]
```diff
@@ -351,7 +351,7 @@ public enum DeploymentType {
       "Config to enable pull queries that scan over the data";
   public static final boolean KSQL_QUERY_PULL_TABLE_SCAN_ENABLED_DEFAULT = true;
 
-  public static final String KSQL_QUERY_STREAM_PULL_QUERY_ENABLED 
+  public static final String KSQL_QUERY_STREAM_PULL_QUERY_ENABLED
```
Comment (kpatelatwork) [inline ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java:354]: nit: unrelated whitespace change to this PR

**Review by kpatelatwork — APPROVED** [review]
LGTM

**Review by mjsax — DISMISSED** [review]

**Review by pbadani — APPROVED** [review]

File: ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java (hunk 1 above)
Comment (VedarthConfluent) [reply ksqldb-common/src/main/java/io/confluent/ksql/util/KsqlConfig.java:354]: It makes the whitespace consistent. Agreed it's unrelated to the PR, but it's a minor improvement, so should be fine.
